### DISCUSSION TITLE
Wazuh agent name (changed) is not reflecting on the Wazuh manager

### DISCUSSION THREAD

**Discussion by subhrajy** [wazuh/wazuh#14849] [Q&A] [answered]
I have recently changed the desktop name of one registered Wazuh agent (windows 11), but it is still showing the old name on the Wazuh manager / dashboard. 

Can anybody help me?
Do Wazuh not detect the names of registered agents?

**Comment by CarlosRS9** [comment, marked as answer]
Hello @subhrajy 

I'm afraid there has been some confusion here.

The only time Wazuh takes the desktop name is during the [agent registration](https://documentation.wazuh.com/current/user-manual/agent-enrollment/index.html) process, and it will only take this name by default when you do not specify another value as your agent's name.

Wazuh currently has a restriction regarding the name of the agents. It cannot be changed once set. The only way to "rename" a registered agent is to delete it and register it again. Therefore, it will not be possible for your Wazuh manager to reflect this change unless you register the agent again.

I hope I have answered your question and sorry for the inconvenience.

Regards

**Reply by subhrajy** [reply]
yeah, thanks for answering. It's an inconvenience that Wazuh does not reflect the changed names, but anyway thanks for clarifying that.
